examples/taproot-psbt.rs

```diff
@@ -0,0 +1,556 @@
+//! Example of taproot PSBT workflow
+
+// We use the alias `alias bt='bitcoin-cli -regtest'` for brevity.
+
+// Step 0 - Wipe the `regtest` data directory to start from a clean slate.
+
+// Step 1 - Run `bitcoind -regtest -daemon` to start the daemon.
+
+// Step 2 - 
+//          2.1) Run `bt -named createwallet wallet_name=benefactor descriptors=true` to create a new descriptor wallet with the name "benefactor"
+//          2.2) Run `bt -named createwallet wallet_name=beneficiary descriptors=true` to create a new wallet with the name "beneficiary"
+//          2.3) Create the two aliases:
+//                  alias bt-benefactor='bitcoin-cli -regtest -rpcwallet=benefactor'
+//                  alias bt-beneficiary='bitcoin-cli -regtest -rpcwallet=beneficiary'
+
+// Step 3 - Run `bt-benefactor generatetoaddress 103 $(bt-benefactor getnewaddress '' bech32m)` to generate 103 new blocks
+//          with block reward being sent to a newly created P2TR address in the `benefactor` wallet.
+//          This will leave us with 3 mature UTXOs that can be spent. Each will be used in a different example below.
+
+// Step 4 -
+//          4.1) Run `bt-benefactor listdescriptors true | jq '.descriptors |  map(select(.desc | startswith("tr("))) | map(.desc)'`
+//               This lists the taproot (tr) private key descriptors for the benefactor wallet. Use the one that ends with the "/86'/1'/0'/0/*"
+//               derivation path. This descriptor is what the wallet uses to derive receive addresses. The other descriptor with
+//               derivation path "/86'/1'/0'/1/*" is used by the wallet to derive change addresses.
+//               Assign the xpriv string to the BENEFACTOR_XPRIV_STR constant below:
+const BENEFACTOR_XPRIV_STR: &str = "tprv8ZgxMBicQKsPdj3TC6RbunTKWQ7oNxYatN3A9pQhv6ZQEzJ2fhnznNwJzYHzLzCcDUrRL4HjVQowWgNANUNGKFq2fWrKC2MqEbjn6YTG1W4";
+//          4.2) Run `bt-beneficiary listdescriptors true | jq '.descriptors |  map(select(.desc | startswith("tr("))) | map(.desc)'`
+//               This lists the taproot (tr) private key descriptors for the benefactor wallet. Use the one that ends with the "/86'/1'/0'/0/*"
+//               derivation path
+//               Assign the xpriv string to the BENEFICIARY_XPRIV_STR constant below:
+const BENEFICIARY_XPRIV_STR: &str = "tprv8ZgxMBicQKsPcyM5XHuyMbMUeyLYpruYpP7BsjiezdG97G1w1GHq8M9dURA2SxtbSiDHTx8ozbru3iEqr41FoAtVUFYNk49jT8tVLg1tiEx";
+
+// Step 5 - Run `bt-benefactor listunspent` to display our three spendable UTXOs and fill them in the constants below:
+const UTXO_1: P2trUtxo = P2trUtxo {
```

Comment:
Perhaps add a comment saying "this is used for example 1", and likewise for the other two utxos.